src/generic/ondemand/field.h

```diff
@@ -0,0 +1,44 @@
+#include "simdjson/error.h"
+
+namespace {
+namespace SIMDJSON_IMPLEMENTATION {
+namespace ondemand {
+
+/**
+ * A JSON field in an field.
+ */
```

Comment:
"A JSON field in an field." is a puzzling description.

Isn't it an abstraction for a key_value_pair?

Reply:
Yes, it looks like I forgot to finish my